packages/vscode-extension/src/panels/Tabpanel.ts

```diff
@@ -60,7 +60,11 @@ export class TabPanel implements Disposable {
   public static render(context: ExtensionContext, fileName?: string, lineNumber?: number) {
     if (TabPanel.currentPanel) {
       // If the webview panel already exists reveal it
-      TabPanel.currentPanel._panel.reveal(ViewColumn.Beside);
+      const activeRNIDEColumn = window.tabGroups.all.find(
+        (group) =>
+          group.activeTab?.label === "React Native IDE" || group.activeTab?.label === "Radon IDE"
+      )?.viewColumn;
+      TabPanel.currentPanel._panel.reveal(activeRNIDEColumn);
```

Comment:
What if `activeRNIDEColumn` will be null? Maybe we should throw an error or warning.

Reply:
Instead of throwing an error lets simply use ViewColumn.Beside when activeRNIDEColumn is null